maintainers/maintainer-list.nix

```diff
@@ -806,6 +806,12 @@
     githubId = 574938;
     name = "Jonathan Glines";
   };
+  austinbutler = {
+    email = "austinabutler@gmail.com";
+    github = "austinbutler";
+    githubId = 354741;
+    name = "Austin Butler";
+  };
```

Comment:
Please make this into a separate commit:

```
maintainers: add <name>
```

Reply:
Done.